modular_skyrat/modules/bluespace_locker/code/game/objects/structures/crates_lockers/bluespace_locker.dm

```diff
@@ -27,6 +27,8 @@
 	icon_state = null
 	desc = ""
 	cutting_tool = null
+	has_opened_overlay = FALSE
+	has_closed_overlay = FALSE
```

Comment:
Does that have any impact on how the bluespace locker will look in-game?

Reply:
It shouldnt due to the locker actively self changing icon state; that simply tells the base procs not to try and grab auto generatred icon states